src/configuration/yaml.md

```diff
@@ -167,3 +141,18 @@ web:
                 '^/sitemap\.xml$':
                     allow: true
 ```
+
+
+### `!archive`
+
+Another custom tag available is `!archive`, which specifies a value is a reference to a directory on disk, relative to the location of the YAML file.  Essentially it defines the value of key as "this entire directory".  Consider this `services.yaml` fragment:
```

Comment:
suggestion: "Essentially it defines the importance of key as "this entire directory".  Consider this `services.yaml` fragment:"

Reply:
Mm, that doesn't fit in English.  Translation issue I think.